primitives/sidechain/src/constants.rs

```diff
@@ -0,0 +1,22 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+//! Sidechain constants
+
+// RPC method names.
+pub const RPC_METHOD_NAME_IMPORT_BLOCKS: &str = "sidechain_importBlock";
+pub const RPC_METHOD_NAME_FETCH_BLOCKS_FROM_PEER: &str = "sidechain_fetchBlocksFromPeer";
```

Comment:
I think this is really only relevant for the worker, this should not be in here.

I would put it in this crate: https://github.com/integritee-network/worker/tree/master/sidechain/rpc-handler